Ntuplizer/python/templates/electronBranches.py

```diff
@@ -34,9 +34,8 @@
         ),
     bools = cms.PSet(
         IsGap = cms.string('isGap'),
-
-        IsCBVIDtight = cms.string('userInt("CBVIDtight")'),
-        IsCBVIDmedium = cms.string('userInt("CBVIDmedium")'),
-        IsCBVIDloose = cms.string('userInt("CBVIDloose")'),
+        IsCBVIDtight = cms.string('userInt("isCBVIDtight")'),
```

Comment:
Oops, I was totally wrong about this, sorry, so please disregard the thing about userFloats and userInts. Code reading fail :(